projects/1.19/assets/l_ender-s-cataclysm/cataclysm/lang/zh_cn.json

```
@@ -134,6 +134,7 @@
   "item.cataclysm.coralssus_spawn_egg": "珊瑚巨像刷怪蛋",
   "item.cataclysm.coral_golem_spawn_egg": "珊瑚傀儡刷怪蛋",
   "item.cataclysm.amethyst_crab_spawn_egg": "紫水晶巨蟹刷怪蛋",
+  "_comment": "争议译名",
   "item.cataclysm.koboleton_spawn_egg": "骷髅狗头人刷怪蛋",
   "item.cataclysm.kobolediator_spawn_egg": "骷髅狗头人将军刷怪蛋",
```

对kobolediator译名的建议请在此提出

---

补充：对 Kobolediator 译作 骷髅狗头人将军 答疑
我认为是 Koboleton + dictator（存疑）的组合，实在是我找不到相关资料了。姑且认为是驱使 Koboleton 行动的“首领”
如有更好译名请提出

---

来自rinko1231的意见：
会不会是 gladiator 角斗士

```suggestion
  "item.cataclysm.koboleton_spawn_egg": "骷髅狗头人角斗士刷怪蛋",
```

似乎不能简单的缩为“斗士”